Tools/CISupport/ews-build/events.py

```diff
@@ -311,8 +311,9 @@ def stopService(self):
 
 
 class GitHubEventHandlerNoEdits(GitHubEventHandler):
-    ACTIONS_TO_TRIGGER_EWS = ('opened', 'synchronize')
+    LABELED = 'labeled'
```

Comment:
this variable name seems weird

Reply:
Would you prefer we reference it as a string literal? Change the name to `LABELED_ACTION`?

Reply:
Just removed the variable entirely.